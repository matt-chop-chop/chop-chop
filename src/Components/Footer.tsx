import React from "react";
import { Flex, Image, Text } from "@chakra-ui/react";
import { siteDescription } from "@/constants";

export const Footer = () => {
  const currentYear: number = new Date().getFullYear();

  return (
    <Flex
      alignItems={["center", "center", "start"]}
      flexDirection="column"
      textAlign={["center", "center", "left"]}
      mt={8}
      p={4}
      width="100%"
    >
      <Image alt="Chop Chop logo" height="64px" src="/images/logo.png" />
      <Text variant="body-small" mt={4} width={["80%", "80%", "40%"]}>
        {siteDescription}
      </Text>
      <Text variant="body-small" mt={4}>
        Â© Chop Chop {currentYear}
      </Text>
    </Flex>
  );
};

export default Footer;
