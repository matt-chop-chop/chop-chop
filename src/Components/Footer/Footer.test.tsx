import { shallow } from "enzyme";
import React from "react";
import Footer from "./Footer";

describe("Footer", () => {
  it("is exported from Footer", () => {
    expect(Footer).toBeDefined();
  });

  it("renders as expected ", () => {
    const wrapper = shallow(<Footer />);

    expect(wrapper).toMatchSnapshot();
  });
});
