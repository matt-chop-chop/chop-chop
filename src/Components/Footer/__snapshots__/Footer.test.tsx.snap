// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders as expected  1`] = `
<Flex
  alignItems={
    [
      "center",
      "center",
      "start",
    ]
  }
  flexDirection="column"
  mt={8}
  p={4}
  textAlign={
    [
      "center",
      "center",
      "left",
    ]
  }
  width="100%"
>
  <Image
    alt="Chop Chop logo"
    height="64px"
    src="/images/logo.png"
  />
  <Text
    mt={4}
    variant="body-small"
    width={
      [
        "80%",
        "80%",
        "40%",
      ]
    }
  >
    Chop Chop is a site designed to allow you to quickly and easily make restaurant quality dishes with ingredients you already have at home.
  </Text>
  <Text
    mt={4}
    variant="body-small"
  >
    Â© Chop Chop 
    2023
  </Text>
</Flex>
`;
